hw/xbox/nv2a/nv2a_psh.c

```diff
@@ -203,7 +211,7 @@ struct PixelShader {
     struct FCInputInfo final_input;
     int tex_modes[4], input_tex[4];
 
-    //uint32_t dot_mapping, input_texture;
+    // uint32_t dot_mapping, input_texture;
```

Comment:
Should probably add a comment that we don't support dotmapping yet (or do we store this thing elsewhere?)
Not even sure what `input_texture` was (or supposed to be) used for.